Example/Core/Tests/FIRLoggerTest.m

```diff
@@ -12,268 +12,14 @@
-  FIRLogWarning(kFIRLoggerCore, kMessageCode, @"%@", self.randomLogString);
-  [[NSRunLoop currentRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:timeInterval]];
-  XCTAssertTrue([self logExists]);
-
-  self.randomLogString = [NSUUID UUID].UUIDString;
-  FIRLogNotice(kFIRLoggerCore, kMessageCode, @"%@", self.randomLogString);
-  [[NSRunLoop currentRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:timeInterval]];
-  XCTAssertTrue([self logExists]);
-
-  // Log messages with Info level and above should NOT be logged to system/device by default.
-  self.randomLogString = [NSUUID UUID].UUIDString;
-  FIRLogInfo(kFIRLoggerCore, kMessageCode, @"%@", self.randomLogString);
-  [[NSRunLoop currentRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:timeInterval]];
-  XCTAssertFalse([self logExists]);
-
-  self.randomLogString = [NSUUID UUID].UUIDString;
-  FIRLogDebug(kFIRLoggerCore, kMessageCode, @"%@", self.randomLogString);
-  [[NSRunLoop currentRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:timeInterval]];
-  XCTAssertFalse([self logExists]);
-#endif
-}
-
-// The FIRLoggerLevel enum must match the ASL_LEVEL_* constants, but we manually redefine
-// them in FIRLoggerLevel.h since we cannot include <asl.h> (see b/34976089 for more details).
-// This test ensures the constants match.
-- (void)testFIRLoggerLevelValues {
-  XCTAssertEqual(FIRLoggerLevelError, ASL_LEVEL_ERR);
-  XCTAssertEqual(FIRLoggerLevelWarning, ASL_LEVEL_WARNING);
-  XCTAssertEqual(FIRLoggerLevelNotice, ASL_LEVEL_NOTICE);
-  XCTAssertEqual(FIRLoggerLevelInfo, ASL_LEVEL_INFO);
-  XCTAssertEqual(FIRLoggerLevelDebug, ASL_LEVEL_DEBUG);
-}
-
-// Helper functions.
-- (BOOL)logExists {
-  [self drainFIRClientQueue];
-  NSString *correctMsg =
-      [NSString stringWithFormat:@"%@[%@] %@", kFIRLoggerCore, kMessageCode, self.randomLogString];
-  return [self messageWasLogged:correctMsg];
-}
-
-- (void)drainFIRClientQueue {
-  dispatch_semaphore_t workerSemaphore = dispatch_semaphore_create(0);
-  dispatch_async(getGULClientQueue(), ^{
-    dispatch_semaphore_signal(workerSemaphore);
-  });
-  dispatch_semaphore_wait(workerSemaphore, DISPATCH_TIME_FOREVER);
-}
-
-- (BOOL)messageWasLogged:(NSString *)message {
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-  aslmsg query = asl_new(ASL_TYPE_QUERY);
-  asl_set_query(query, ASL_KEY_FACILITY, kGULLoggerASLClientFacilityName, ASL_QUERY_OP_EQUAL);
-  aslresponse r = asl_search(getGULLoggerClient(), query);
-  asl_free(query);
-  aslmsg m;
-  const char *val;
-  NSMutableArray *allMsg = [[NSMutableArray alloc] init];
-  while ((m = asl_next(r)) != NULL) {
-    val = asl_get(m, ASL_KEY_MSG);
-    if (val) {
-      [allMsg addObject:[NSString stringWithUTF8String:val]];
-    }
-  }
-  asl_free(m);
-  asl_release(r);
-  return [allMsg containsObject:message];
-#pragma clang pop
-}
-
+// TODO(bstpierre): Test FIRLogger functionality separate from GULLogger.
```

Comment:
Is anything going from tested to untested with this move?

Reply:
Added #2423 to track this testing.